tutoraspects/templates/aspects/apps/superset/pythonpath/create_assets.py

```diff
@@ -248,31 +248,34 @@ def import_assets():
     # performance_metric script however, so we keep them in the assets.
     # This just blanks them in the database after import, which forces a
     # query to get the assets instead of using the query context.
+    owners = get_owners()
```

Comment:
These changes seem odd since users with the same username may exist across systems and might cause permissions issues with the wrong people being assigned permissions. What is this change trying to accomplish?

Reply:
Faciliate development by making developers owners of the charts. This shouldn't be used in production

Reply:
Yeah, it's fine. I thought it would use a default user, but by default SUPERSET_OWNERS is empty so someone would have to opt in to use this. Is that correct?

Reply:
Yep